### PR TITLE
[119] Update support playbook

### PR DESCRIPTION
**Pull request by goodviber**
### Context
Instructions for transfer of courses to another provider
### Changes proposed in this pull request
Update docs with instructions
### Guidance to review
Trello card [505](https://trello.com/c/SyjMR5lT/505-update-support-playbook)
### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by goodviber** [comment]
[Support](https://trello.com/c/HOrLOAgK/119-support)

**Comment by goodviber** [comment]
[Update Support Playbook](https://trello.com/c/SyjMR5lT/505-update-support-playbook)

File: docs/support_playbook.md [hunk 1]
```diff
@@ -63,6 +63,20 @@ To change the accrediting body of a course, you can do the following:
 p = RecruitmentCycle.current.providers.find_by(provider_code: "1YP")
 p.courses.update(accredited_body_code: "1YK")
 ```
+## Transfer courses to another provider
+
+```ruby
+# This can be reviewed in QA before actioning in production
```
Comment (avinhurry) [inline docs/support_playbook.md:69]: Do you mean tested/tried in QA before trying in prod? I find the wording a bit confusing.

**Review by avinhurry — APPROVED** [review]
Nice one. I had one question. Other than that looks good 👍🏼

File: docs/support_playbook.md (hunk 1 above)
Comment (goodviber) [reply docs/support_playbook.md]: I have updated the wording, thanks

**Review by defong — APPROVED** [review]

**Pull request merged**